examples/others/easings.h

```diff
@@ -107,17 +107,17 @@ EASEDEF float EaseLinearOut(float t, float b, float c, float d) { return (c*t/d
 EASEDEF float EaseLinearInOut(float t,float b, float c, float d) { return (c*t/d + b); }
 
 // Sine Easing functions
-EASEDEF float EaseSineIn(float t, float b, float c, float d) { return (-c*cosf(t/d*(PI/2.0f)) + c + b); }
-EASEDEF float EaseSineOut(float t, float b, float c, float d) { return (c*sinf(t/d*(PI/2.0f)) + b); }
-EASEDEF float EaseSineInOut(float t, float b, float c, float d) { return (-c/2.0f*(cosf(PI*t/d) - 1.0f) + b); }
+EASEDEF float EaseSineIn(float t, float b, float c, float d) { return (-c*(float)cosf(t/d*(PI/2.0f)) + c + b); }
```

Comment:
`cosf()` and `sinf()` already return float values, no need to cast.

Reply:
Related to PI change and missed reverts.